sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/eventhubs/AzureEventHubProperties.java

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.eventhubs;
+
+import com.azure.messaging.eventhubs.LoadBalancingStrategy;
+import com.azure.messaging.eventhubs.models.EventPosition;
+import com.azure.spring.cloud.autoconfigure.storage.blob.AzureStorageBlobProperties;
+import com.azure.spring.core.properties.AzurePropertiesUtils;
+import org.springframework.beans.BeanUtils;
+import org.springframework.boot.context.properties.PropertyMapper;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Azure Event Hub related properties.
+ */
+public class AzureEventHubProperties extends AzureEventHubConsumerProperties {
```

Comment:
this hierarchy looks wired?

Reply:
This will make the merge properties easier.